api-reference/50 Common/utils/excelExporter/exportDataGrid(options).md

```diff
@@ -14,7 +14,7 @@ Export settings.
 
 ---
      
-This method requires <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> to export data and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> to save files.
+This method requires <a href="https://github.com/exceljs/exceljs" target="_blank">ExcelJS</a> v3.3.1 or newer to export data and <a href="https://github.com/eligrey/FileSaver.js/" target="_blank">FileSaver</a> to save files.
 
 You can call this method at any point in your application. In the example below, this method is called in the [onExporting](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/onExporting.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting') function that is executed before data is exported. The **cancel** parameter is enabled to prevent the built-in export. As a result, the **DataGrid** is exported as is to a single worksheet. 
```

Comment:
"The **cancel** parameter is enabled to prevent the built-in export."
Remove the article ("the") before "built-in export"
"As a result, the **DataGrid** is exported as is to a single worksheet."
Do you mean: "As a result, the **DataGrid** is exported to a single worksheet file."

Also, what is the text in Line 20 below? Is it a typo?

Reply:
Not really. The worksheets(at least in terms of this library) are the tabs int he bottom of the excel window: 
![image](https://user-images.githubusercontent.com/33040934/78221874-e0175f80-74cc-11ea-9d3c-83e91912b6ae.png)

Separate files are called 'workbooks'.

Reply:
In that case, please revise it as follows:
"As a result, the DataGrid is exported to a single worksheet."